lib/signs_ui_web/router.ex

```diff
@@ -9,7 +9,7 @@ defmodule SignsUiWeb.Router do
 
     plug(:put_secure_browser_headers, %{
       "content-security-policy" =>
-        "default-src 'self'; script-src 'self' 'unsafe-inline' 'unsafe-eval'; style-src 'self' 'unsafe-inline'"
+        "default-src 'self'; connect-src 'self' #{Application.get_env(:sentry, :dsn, "")}; script-src 'self' 'unsafe-inline' 'unsafe-eval'; style-src 'self' 'unsafe-inline'"
```

Comment:
Does this mess anything up if the DSN isn't defined?

Reply:
It shouldn't, that's why I added `""` as a default. Open to other suggestions here, I thought about extracting this into a function but that seemed too heavy.

Reply:
Oh, I see; `'self'` is a reference to the current host, and DSN is the host we're connecting to? And if there's no DSN, the line looks like `connect-src 'self'  ;` and the extra space doesn't affect anything?

Reply:
Yes.

Reply:
No, the extra space seems okay. It's okay for `connect-src` to have a full URL and not only a domain name?

Reply:
Hmmm. well, it seemed to work with the full URL but looking at the docs it seems it's only supposed to be the host. the host is different per DSN though so we would need to either extract it from the variable or use a wildcard. consequences of using the wildcard are not extreme, it would just mean that connecting to any DSN would be allowed.

Reply:
Maybe `*.ingest.sentry.io`? That seems like it would reduce the blast radius pretty far.

Reply:
That was my thought.